src/Time.coffee

```diff
@@ -1,14 +1,18 @@
-
 class CountDown
+
   constructor: ({@time = 120, @delay = 1000}) ->
     @onComplete = ->
     @onTick = ->
+    @pauseTimer = false
```

Comment:
instead of introducing a random property to the class, extending it's public API with `.pause()` and `.resume()` methods would be cleaner